tests/testthat/_snaps/bump_version.md

```diff
@@ -55,3 +95,7 @@
     x Must be on the main branch (main) for running fledge functions.
     i Currently on branch bla.
 
+# bump_version() errors well for wrong arguments
```

Comment:
This reminds me of #153 (the error for the which argument looks different at the moment)